fix(curriculum): changed description for anchor tag in learn-html-by-building-a-cat-photo-app (#50616)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md

# --description--

> Enclose below text `freeCodeCamp.org` with an anchor (`a`) element to make it a link and set the `href` attribute of `a` element to `https://www.freecodecamp.org`.

Reviewer: ```suggestion
Turn the existing `freeCodeCamp.org` text into a link by enclosing it in an anchor (`a`) element. The `href` attribute should be set to `https://www.freecodecamp.org`."
```

I think your new wording is slightly awkward. My suggestion is @moT01 's wording from the issue.

Author: ok. I'll add @moT01 's wording.

> Turn the existing `freeCodeCamp.org` text into a link by enclosing it in an anchor (`a`) element. The `href` attribute should be set to `https://www.freecodecamp.org`."

Author: Sorry sorry! I didn't see that

# --hints--

> Your anchor (`a`) element should be nested within the `p` element. Make sure to add an opening tag and closing tag for the anchor (`a`) element.

Reviewer: ```suggestion
Your anchor (`a`) element should be nested within the `p` element inside the `footer`. Make sure to add an opening tag and closing tag for the anchor (`a`) element.
```

There are multiple `p` elements in the HTML so we should specify which one.
